### DISCUSSION TITLE
notification email if a reaction and/or sample and/or collection gets published

### DISCUSSION THREAD

**Discussion by tilfischer** [ComPlat/chemotion_REPO#89] [Ideas]
Dear all,

If users want to publish reactions or samples via the "Fly" button, they include analyses, select authors, references and may add additional reviewers. They also need to check a bock telling "I know that the data that is submitted without the selection of an embargo bundle will be published immediately after a successful review" and agrees with this workflow. 

What I would, anyhow, expect is that the users receive a notification email if one of their reactions and/or samples made it though the review and gets published. This is currently not the case, but might be implemented. 

For many reactions and samples, possibly corresponding to research results (lets call it study (or investigation?)) reported in an article, users might have selected to create an embargo bundle. As a matter of course, users should only receive a single notification email when their embargo bundle is released, rather than a flood of hundreds of emails, one for each reaction and/or sample.

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer 

The notification email will be sent to the contributor once the reactions/samples have been reviewed and published, or when the embargo bundle is released. Could you please provide the publication link?

Best regards,
Claire

**Comment by tilfischer** [comment]
Could you explain further? I don't understand what you mean by "publication link"?

**Comment by cllde8** [comment]
Dear @tilfischer 

Apologies for any confusion. Could you please provide the DOI for which you mentioned not receiving the notification email? Thank you.

Best regards,
Claire

**Comment by tilfischer** [comment]
https://dx.doi.org/10.14272/UVLHGRADYISRGZ-UHFFFAOYSA-N.11

**Comment by tilfischer** [comment]
> The notification email will be sent to the contributor once the reactions/samples have been reviewed and published, or when the embargo bundle is released. Could you please provide the publication link?

How about authors/creators? They also might want to be notified if a reaction or sample or bundle gets published.

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer 

Thank you for your prompt response. Just to clarify, when this sample is published, the notification email is sent to the contributor, Alrik Dorn in this case. Currently, authors/creators won't receive the notification email. We understand that some authors/creators may prefer to receive the notification email while others may not. I believe this is open to discussion.

Best regards,
Claire

**Comment by tilfischer** [comment]
Dear Claire,

I totally agree. 

Best,
Tillmann

**Comment by cllde8** [comment]
Dear @tilfischer 

Thanks for the reply. I will convert it to Discussion.

Best regards,
Claire
